### ISSUE TITLE
NTR, pulmo: interstitial emphysema

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5579]
**Preferred term label:**
interstitial emphysema

**Synonyms**

**Definition (free text, please give PubMed ID)**
Interstitial emphysema is characterized by air dissecting within the interstitium of the lung, typically in the peribronchovascular sheaths, interlobular septa, and visceral pleura. It is most commonly seen in neonates receiving mechanical ventilation. It is rarely recognized radiographically in adults and is infrequently seen on CT scans. It appears as perivascular lucent or low attenuating halos and small cysts

PMID: 18195376

**Parent term (use hpo.jax.org/app)**
Emphysema, HP:0002097

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
LMU crapp, @mgriese, @katrinknoflach

**Comment by pnrobinson** [comment]
adding term as HP:0032965
